ui/design-system-core/src/components/Icon/index.tsx

```diff
@@ -1,16 +1,27 @@
 import React from 'react';
 import * as HeroIcons from '@heroicons/react/24/outline';
+import * as SolidHeroIcons from '@heroicons/react/24/solid';
 
+export type icontype = 'solid' | 'outline';
 export type IconName = keyof typeof HeroIcons;
-interface IconProps {
+export interface IconProps {
   icon: IconName;
   styling?: string;
+  strokeColor?: string;
+  fillColor?: string;
+  iconType?: icontype;
```

Comment:
The Icon interface has changed, so you could update with next and resolve the changes

Reply:
Thanks. I have modified it.